docker/compose/mozdef_meteor/Dockerfile

```diff
@@ -39,12 +44,12 @@ RUN chown -R mozdef:mozdef /opt/mozdef/envs/mozdef/meteor
 
 USER mozdef
 RUN \
-  mkdir -p /opt/mozdef/envs/meteor/mozdef && \
-  cd /opt/mozdef/envs/mozdef/meteor && \
-  meteor npm install && \
-  meteor build --server localhost:3002 --directory /opt/mozdef/envs/meteor/mozdef && \
-  mv /opt/mozdef/envs/mozdef/meteor/node_modules /opt/mozdef/envs/meteor/mozdef/node_modules
+  mkdir -p /opt/mozdef/envs/meteor/mozdef
+  #cd /opt/mozdef/envs/mozdef/meteor && \
```

Comment:
Same here, can we nuke these commented out lines?

Reply:
Since those build the meteor package to run with node, I added an option to the live reload overlay to trigger whether or not that step happens (will happen by default). 
https://github.com/mozilla/MozDef/pull/770/commits/efc8d2f95f0e0869f4f360ff42b8652c5abcce29